docs/source/use_cases/index.rst

```diff
@@ -13,6 +13,7 @@ Below is a gallery of use cases.
    /use_cases/face_mask_detection.md
    /use_cases/group_size_checking.md
    /use_cases/object_counting.md
+   /use_cases/object_tracking.md
```

Comment:
I think this should be `/use_cases/multi_object_tracking.md` instead to match the document name

Reply:
You're right, updated accordingly